editor/src/components/inspector/common/css-utils.ts

```diff
@@ -4427,6 +4429,7 @@ export const cssParsers: CSSParsers = {
   width: parseCSSLengthPercent,
   height: parseCSSLengthPercent,
   flexBasis: parseCSSLengthPercent,
+  zIndex: parseCSSLengthPercent,
```

Comment:
I think this could be `parseCSSUnitless`

Reply:
great idea! updated in https://github.com/concrete-utopia/utopia/pull/3668/commits/67d6378b9252e628a029c27ccb943c62e458ad41